src/apis/medTechApi.ts

```diff
@@ -190,10 +192,15 @@ export class MedTechApiBuilder {
     return this;
   }
 
-  build() : MedTechApi {
-    const api = Api(this.iCureBasePath!, this.userName!, this.password!, this.crypto);
+  preventCookieUsage(): MedTechApiBuilder {
```

Comment:
We can put a default value but this should be provided by the user

Reply:
I don't know why, but this is different from the code I have, that provides the parameter. Probably I didn't do a rebase